blogs/ecs-ec2-crossvpc-with-tls/mesh.sh

```diff
@@ -0,0 +1,232 @@
+# Mesh components for Crystal backend
```

Comment:
It looks like there's a CloudFormation template for these, is this necessary?

Reply:
removed the shell script